index.client.js

```diff
@@ -56,17 +56,28 @@ ReactI13nGoogleAnalytics.prototype.getPlugin = function () {
              * @param {Function} calback callback function
              */
             pageview: function (payload, callback) {
+                var args = {},
+                    argsKey = ['location', 'page', 'title'];
+
+                for (i in argsKey) {
+                    var prop = argsKey[i];
+                    if (payload.hasOwnProperty(prop)) {
```

Comment:
I should rephrase more clearly about the PR story~
We still let user to able to **assign** key even the value is undefined.  
Only when the key does  not exist, we left ga to determine the its default value.
So it indeed what I expect